chain-abci/src/app/mod.rs

```diff
@@ -196,7 +196,10 @@ impl abci::Application for ChainNodeApp {
             kvpair.key = Vec::from(&b"txid"[..]);
             // TODO: "Keys and values in tags must be UTF-8 encoded strings" ?
             kvpair.value = Vec::from(&txaux.tx_id()[..]);
-            resp.tags.push(kvpair);
+            let mut event = Event::new();
+            event.field_type = "delivered_txs".to_string();
```

Comment:
I guess these field types can be anything or at least I didn't find any convention: https://tendermint.com/docs/spec/abci/abci.html#events ?

perhaps extra these strings / event types into constants -- I'm assuming they may need to be used in client, so perhaps put them in chain-core (could be in a separate PR)?

Reply:
And as there are now more key-value levels, the event type could be named based on what it contains: e.g. "valid_txs" in this case, "block_filter" in the other?

Reply:
Extracted these event types in an enum in `chain_core`